src/test/java/com/google/devtools/build/lib/remote/BuildWithoutTheBytesIntegrationTest.java

```diff
@@ -360,6 +360,292 @@ public void changeOutputMode_invalidateActions() throws Exception {
     }
   }
 
+  @Test
+  public void symlinkToSourceFile() throws Exception {
```

Comment:
Add a test case that symlink are not created on the local file system if no local actions need them?

Reply:
I believe this is obsoleted by the fact that `RemoteActionFileSystem#createSymbolicLink` now creates the symlink on both filesystems. This seems unavoidable since `RemoteActionFileSystem` is also used in contexts where modifying the local filesystem is expected (it might be possible to move this to a different filesystem implementation, but not today).